lxd/api_project.go

```diff
@@ -307,7 +307,7 @@ func projectsPost(d *Daemon, r *http.Request) response.Response {
 	}
 
 	// Quick checks.
-	err = projectValidateName(project.Name)
+	err = projecthelpers.ValidName(project.Name)
```

Comment:
Although not a change you've introduced here, a note for future change is to stop using the `projecthelpers` import alias and just use `project` like everywhere else in the code base, and then change any shadowing variables in this file to use `p` rather than `project`. 

This is for consistency and helps with code searches.

Reply:
Noted for the next time. Thanks

Reply:
I'm happy to bundle that `unshadowing` change so that we can avoid the `projecthelpers`.

Reply:
cool thanks

Reply:
Hmm, can I backpedal on that one? This being in `lxd/api_project.go`, there are more than 200 occurrences of "project".